go/worker/keymanager/worker.go

```diff
@@ -486,68 +603,36 @@ func (w *Worker) worker() { // nolint: gocyclo
 				)
 				continue
 			}
+			// New runtimes can be allowed with the policy update.
+			if err = w.recheckAllRuntimes(currentStatus); err != nil {
+				w.logger.Error("failed rechecking runtimes",
+					"err", err,
+				)
+				return
```

Comment:
Should this `continue` instead of terminating the whole key manager worker?

Reply:
Yeah maybe that's more in line with the rest of the code. Although we do `return` already if the `WatchEvents` fails (line 596), and similarly i made the `recheckAllRuntimes` return an error only in case the registry query fails. But probably still better to `continue` here so will change it.